lib/action_view/component/base.rb

```diff
@@ -40,6 +40,8 @@ class Base < ActionView::Base
       # <span title="greeting">Hello, world!</span>
       #
       def render_in(view_context, *args, &block)
+        return '' unless render?
```

Comment:
```suggestion
        return "" unless render?
```

There is a Rubocop linting issue here that is causing the checks to fail.

Reply:
Thanks for catching that! (For some reason my editor is silently throwing an error running Rubocop, which I guess is why I missed this).